tee-worker/client-api/sidechain-api/package.json

```diff
@@ -5,7 +5,7 @@
     "main": "dist/src/index.js",
     "module": "dist/src/index.js",
     "sideEffects": false,
-    "version": "0.0.2-next.8",
+    "version": "0.0.2-10",
```

Comment:
not sure on this versioning. what do you think, @0xverin?

Reply:
Should we keep consistent with [identity-worker](https://hub.docker.com/layers/litentry/identity-worker/v0.9.18-10/images/sha256-09126086f30f298b952a3c9a3cde53b945148703bf982a7b7a56dcf71f767263?context=explore) now? Since parachain are now only using the node version to manage the versions of other workers, I suggest we also keep consistent with them here.

Reply:
do you mean making it v0.9.18-10 too?

Reply:
Yep.Now 0.02 is meaningless, I think.

Reply:
sounds good to me!

Reply:
done